### DISCUSSION TITLE
Mismatched network.

### DISCUSSION THREAD

**Discussion by TECHBABSY** [blockscout/blockscout#11007] [General]
I mistakenly transfered 10 dollar worth of DC ( Dogechain) token from mexc to uniswap.  Unfortunately,  uniswap doesn't support dogechain and my funds never dropped.  Please how can I recover my funds into Mexc ?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
